skimage/_shared/_default_value.py

```diff
@@ -0,0 +1,14 @@
+class DefaultValue(object):
```

Comment:
Use Ellipsis instead of None?

Reply:
Good suggestion, 👍

Reply:
@stefanv @soupault to clarify, you mean, get rid of this object, and just use Ellipsis directly?

Reply:
As I see it, use Ellipsis instead of 'None' in this class. Your option would work equally well, though